src/utils/cache.js

```diff
@@ -0,0 +1 @@
+module.exports = new Map();
```

Comment:
@michaelmurray6298 please confirm that you are using this cache (I haven't seen it used yet). If so, great! If not, please remove this file.